content/en/blog/2023/traffic-for-ambient-and-sidecar/index.md

```diff
@@ -21,8 +21,6 @@ To clarify the network traffic paths and make it easier to understand, this blog
 
 _Note 1: The following analysis is based on Istio 1.18.2, where ambient mode uses iptables for redirection._
```

Comment:
If we are really removing note 2, then the 1 could be removed here, and maybe make it an info? section.